src/logging.cpp

```diff
@@ -162,6 +162,7 @@ const CLogCategoryDesc LogCategories[] =
     {BCLog::COINDB, "coindb"},
     {BCLog::QT, "qt"},
     {BCLog::LEVELDB, "leveldb"},
+    {BCLog::VALIDATIONINTERFACE, "validationinterface"},
```

Comment:
In commit "Add VALIDATIONINTERFACE to BCLog::LogFlags" (389d423a252e43d0b08871f8ae5b66a84a8a217c)

Would change names to `VALIDATION, "validation"`. These are just validation events. Fact that they are printed in the validationinterface source file instead of the validation one is an implementation detail that could change and would be arcane to someone just trying to configure logging categories.